src/linter/ui5Types/fix/collections/sapUiCoreFixes.ts

```diff
@@ -28,6 +78,75 @@ t.declareModule("sap/ui/core/Configuration", [
 				return `new ${identifierNames[0]}(${identifierNames[1]}.getLanguageTag())`;
 			},
 		})),
+		t.method("getRTL", accessExpressionFix({
+			moduleName: "sap/base/i18n/Localization",
+			propertyAccess: "getRTL",
+		})),
+		t.method("getSAPLogonLanguage", accessExpressionFix({
+			moduleName: "sap/base/i18n/Localization",
+			propertyAccess: "getSAPLogonLanguage",
+		})),
+		t.method("getSecurityTokenHandlers", accessExpressionFix({
+			moduleName: "sap/ui/security/Security",
+			propertyAccess: "getSecurityTokenHandlers",
+		})),
+		t.method("getTheme", accessExpressionFix({
+			moduleName: "sap/ui/core/Theming",
+			propertyAccess: "getTheme",
+		})),
+		t.method("getTimezone", accessExpressionFix({
+			moduleName: "sap/base/i18n/Localization",
+			propertyAccess: "getTimezone",
+		})),
+		t.method("getUIDPrefix", accessExpressionFix({
+			moduleName: "sap/ui/base/ManagedObjectMetadata",
+			propertyAccess: "getUIDPrefix",
+		})),
+		t.method("getWhitelistService", accessExpressionFix({
+			moduleName: "sap/ui/security/Security",
+			propertyAccess: "getAllowlistService",
+		})),
+		t.method("setAnimationMode", accessExpressionFix({
+			moduleName: "sap/ui/core/ControlBehavior",
+			propertyAccess: "setAnimationMode",
+		})),
+		t.method("setCalendarType", callExpressionFix({
+			moduleName: "sap/base/i18n/Formatting",
+			propertyAccess: "setCalendarType",
+			mustNotUseReturnValue: true,
+		})),
+		t.method("setCalendarWeekNumbering", callExpressionFix({
+			moduleName: "sap/base/i18n/Formatting",
+			propertyAccess: "setCalendarWeekNumbering",
+			mustNotUseReturnValue: true,
+		})),
+		t.method("setFormatLocale", callExpressionFix({
+			moduleName: "sap/base/i18n/Formatting",
+			propertyAccess: "setLanguageTag",
+			mustNotUseReturnValue: true,
+		})),
+		t.method("setSecurityTokenHandlers", accessExpressionFix({
+			moduleName: "sap/ui/security/Security",
+			propertyAccess: "setSecurityTokenHandlers",
+		})),
+		t.method("setTheme", callExpressionFix({
+			moduleName: "sap/ui/core/Theming",
+			propertyAccess: "setTheme",
+			mustNotUseReturnValue: true,
+		})),
+		t.method("setTimezone", callExpressionFix({
+			moduleName: "sap/base/i18n/Localization",
+			propertyAccess: "setTimezone",
+			mustNotUseReturnValue: true,
+		})),
+		...t.methods(["setRTL", "setLanguage"], callExpressionFix({
+			scope: FixScope.FirstChild,
```

Comment:
Just fyi:

Yesterday I noticed the scope was wrong here. I corrected it in https://github.com/SAP/ui5-linter/commit/b08b237866addf0f31578eabd81771fe5110725b#diff-0b1c00da90d25004cd6d18142125bea5ced9fd4708ea7ac9a5d7519ac151960fR17

You'll probably see this in a conflict when rebasing this PR 👍